private/manageFishData.py

```diff
@@ -443,6 +460,7 @@ def _convert_fish_to_json(item):
                          'patch': item.get('patch'),
                          'folklore': folklore,
                          'collectable': is_collectable,
+                         'minRating': min_rating,
```

Comment:
Slightly concerned that this could be misinterpreted as "min gathering rating".  Would it work to just have `collectable` be either an int, or null (None)?  In the JavaScript code, it's easy enough to test for this: `collectable !== null` meaning it is collectable, and the value represents the min rating.

Reason for concern is other projects such as Cactbot actually are using my data.js file... even though they never actually contacted me... I'd like to ensure my data format is as intuitive as possible, and know I prolly only get one chance to add anything hahaha.

Reply:
Merged collectable rating threshold into collectable field.